mingw-w64-python-jaraco.itertools/PKGBUILD

```
@@ -0,0 +1,44 @@
+# Maintainer: ???
+
+_pyname=jaraco.itertools
+_realname=jaraco.itertools
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+pkgver=6.0.3
+pkgrel=1
+pkgdesc='Jaraco Itertools'
```

```suggestion
pkgdesc='Jaraco Itertools (mingw-w64)'
```

Or something more descriptive (I failed to find one)